scripts/viz/tsne_viz.R

```diff
@@ -1,25 +1,47 @@
 # Pan-Cancer Variational Autoencoder
 # Gregory Way 2017
 #
-# Visualize t-SNE plots of VAE and raw RNAseq data features
+# Visualize t-SNE plots of VAE, ADAGE, and raw RNAseq data features
 
```

Comment:
Add version of R used since there is some versioning issues

Reply:
Great point. I realize now that I don't have a conda environment for R scripts!! I will need to add one, but it is beyond the scope of this pull request.